src/middlewares/paramsValidate.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * @param {*} schema
+ * @returns{object} allows next() if the request body is valid otherwise returns an error
+ */
+export default function paramsValidate(schema) {
+  return (req, res, next) => {
+    const { error } = schema.validate(req.params);
+    if (error) {
+      return res.status(400).send({
+        status: 400,
+        data: { error: error.details[0].message.replace(/[/"]+/g, '') }
+      });
+    }
+    next();
+  };
+}
```

Comment:
You will have also to return more than just error message, return the whole error object also.